examples/trials/mnist-advisor/config_pai.yml

```diff
@@ -27,10 +27,13 @@ trial:
   memoryMB: 8196
   #The docker image to run nni job on pai
   image: msranni/nni:latest
+  nniManagerNFSMountPath: /home/user/mnt
+  containerNFSMountPath: /mnt/data/user
+  paiStoragePlugin: team_wise
 paiConfig:
   #The username to login pai
   userName: username
-  #The password to login pai
-  passWord: password
+  #The token to login pai
+  token: token
```

Comment:
do we have a description of "where could I get the token"?

Reply:
It's like github access token. It's achievable from PAI user settings. I think users can figure that out.

Reply:
the token tutorial is in https://github.com/microsoft/nni/pull/2011